public/js/ball.js

```diff
@@ -17,11 +17,19 @@ class Ball {
         this.resetPosition();
     }
 
+    #generateStartAngleRad() {
```

Comment:
what's this hashtag at the start mean?

Reply:
Private class members: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields